### PR TITLE
dev-util/cppcheck: version bump to 1.82 with github SRC_URI

### PR DESCRIPTION
**Pull request by realnc**
Missing "|| die" statements added where needed.
Build log: https://bugs.gentoo.org/attachment.cgi?id=523844

Closes: https://bugs.gentoo.org/646608
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	# Drop bundled libs, patch Makefile generator and re-run it
+	rm -r externals/tinyxml || die
+	tc-export CXX
+	emake dmake
+	./dmake || die
+}
+
+src_configure() {
+	if use pcre ; then
+		sed -e '/HAVE_RULES=/s:=no:=yes:' \
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: missing `|| die`

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+src_compile() {
+	export LIBS="$(pkg-config --libs tinyxml2)"
+	emake ${PN} man \
+		CFGDIR="${EROOT}usr/share/${PN}/cfg" \
+		DB2MAN="${EROOT}usr/share/sgml/docbook/xsl-stylesheets/manpages/docbook.xsl"
+
+	if use qt5 ; then
+		pushd gui
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: missing `|| die`

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+		eqmake5
+		emake
+		popd
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: missing `|| die`

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	fi
+	if use htmlreport ; then
+		pushd htmlreport
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: Same as above.

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+		distutils-r1_src_compile
+		popd
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: Same as above.

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+	fi
+}
+
+src_test() {
+	# safe final version
+	mv -v ${PN}{,.final}
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: All those `mv` commands need to `|| die`

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+	emake install DESTDIR="${ED}"
+
+	insinto "/usr/share/${PN}/cfg"
+	doins cfg/*.cfg
+	if use qt5 ; then
+		dobin gui/${PN}-gui
+		dodoc gui/{projectfile.txt,gui.${PN}}
+	fi
+	if use htmlreport ; then
+		pushd htmlreport
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: Same as above.

File: dev-util/cppcheck/cppcheck-1.82.ebuild [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+		distutils-r1_src_install
+		popd
```
Comment (monsieurp) [inline dev-util/cppcheck/cppcheck-1.82.ebuild]: Same as above.

**Comment by realnc** [comment]
@monsieurp What is the review policy for version bumps? Are people expected to fix the ebuilds and make them perfect? Because I did not write this. I just copied it to a new filename.

**Comment by monsieurp** [comment]
Hi @realnc. We should avoid copying and pasting errors all over the place and if we spot one, we should fix it. Even for trivial version bumps, we should strive for quality and not quick'n'dirty copy paste. Do you agree?

**Comment by realnc** [comment]
@monsieurp I don't disagree. Was just asking so I know if I can submit a version bump for something without having to fix everything in the current ebuild first.

**Comment by realnc** [comment]
@monsieurp Is there anything else needed here?

**Comment by realnc** [comment]
@xmw Can you merge?

**Pull request closed without merging**